### PR TITLE
refactor(jest): use it.each & remove sourcemaps from snapshots

### PR DESCRIPTION
**Pull request by wardpeet**
While working on the bublé => babel conversion the source-maps snapshot was actually cluttering up the diffs and I'm unsure why they are there. I changed the tests to omit the source-maps from snapshots but  I do check file count to make sure they are created.

### CODE REVIEW

File: test/index.test.js [hunk 1]
```diff
@@ -55,64 +55,70 @@ const parseScript = (() => {
 })();
 
 describe('fixtures', () => {
+	const dirs = [];
 	fs.readdirSync(FIXTURES_DIR).forEach(fixtureDir => {
```
Comment (ForsakenHarmony) [inline test/index.test.js]: this could probably be a map + filter for a more functional approach

File: test/index.test.js [hunk 2]
```diff
@@ -55,64 +55,70 @@ const parseScript = (() => {
...
-						'Directory tree:',
-						printedDir,
-						strip(output),
-					].join('\n\n'),
-				).toMatchSnapshot();
+		dirs.push(fixtureDir);
+	});
 
-				fs.readdirSync(resolve(dist)).forEach(file => {
+	it.each(dirs)(
```
Comment (ForsakenHarmony) [inline test/index.test.js]: didn't know this was a thing 👌

File: test/index.test.js (hunk 1 above)
Comment (wardpeet) [reply test/index.test.js]: 👍  make sense!

**Comment by ForsakenHarmony** [comment]
nice

**Pull request merged**